tests/bdd/zfs.feature

```diff
@@ -0,0 +1,87 @@
+Feature: Validate volume provisioning for fsType zfs, ext4, xfs, btrfs
+         Validate snapshot creation and clone
+         Validate volume resize
+
+  Background:
+    Given a single zfs cluster is configured
+    And zfs pool pool created on the node
+
+  Scenario: volume provision for file system type zfs, ext4, xfs, btrfs
+    Given a storage class is created with fsType as zfs, ext4, xfs, btrfs respectively 
+    And pvc is created referencing the same sc
+    When a deployment is created using the same pvc
+    Then zfsvolume must be created and used by the application
+
+  Scenario: volume property change for file system type zfs, ext4, xfs, btrfs
+    Given a storage class is created with fsType as zfs, ext4, xfs, btrfs respectively 
+    And pvc is created referencing the same sc
+    And a deployment is created using the same pvc
+    When the zfsvolume properties like compression, dedep are updated
+    Then the zfsvolume properties must reflect the updated value
+
+  Scenario: volume property change for file system type zfs, ext4, xfs
+    Given a storage class is created with fsType as zfs, ext4, xfs, btrfs respectively 
+    And pvc is created referencing the same sc
+    And a deployment is created using the same pvc
+    When the zfsvolume volume type is DATASET and record size is updated
+    Then the record size of must show the updated value of zfsvolume
+
+  Scenario: volume property change for file system type zfs, ext4, xfs
```

Comment:
We cannot use same scenario name for all the cases. Please change it to meaningful ones.